webcompat/form.py

```diff
@@ -100,6 +114,15 @@ def get_problem(category):
     return u'Unknown'
 
 
+def get_browser_test(category):
```

Comment:
It's better to have different names for different things. And probably starting with the test will help clarify what we really need to achieve with the function. Try as much as possible to write your test first that will fail. Then write your function to fulfill the tests. It helps a lot to have a clearer idea. 

if two functions are really doing the same thing, then it's only one function which should be versatile enough to handle the two cases to avoid repeating code. In the current form of the code, it looks like only one function. But probably we need to wait for @zoepage answer on the design Radio field or checkbox.